net/bluetooth/bluetooth_sockif.c

```diff
@@ -439,6 +439,7 @@ static int bluetooth_l2cap_bind(FAR struct socket *psock,
                                 FAR const struct sockaddr_l2 *iaddr,
                                 socklen_t addrlen)
 {
+  FAR struct socket_conn_s *sconn = psock->s_conn;
```

Comment:
should we change all sconn to conn to unity the term?

Reply:
And let's add DEBUGASSERT(conn != NULL).

Reply:
> And let's add DEBUGASSERT(conn != NULL).

done

> should we change all sconn to conn to unity the term?

I think sconn is better, “conn” is a common naming in most files

Reply:
> > And let's add DEBUGASSERT(conn != NULL).
> 
> done
> 
> > should we change all sconn to conn to unity the term?
> 
> I think sconn is better, “conn” is a common naming in most files

Otherwise, there is a confusion with tcp_conn_s.sconn field that appears now in many places. I think conn is better when psock->s_conn is referred.